contributors/distribution-guidelines.md

```diff
@@ -1,17 +1,64 @@
 # Guidelines for Distributions
 
-So you want to distribute your own version of a Chef product:
+So you want to create a new distro of a Chef product….
 
-Congratulations! We welcome distributions of any and all of our products with open arms.
+Excellent! Welcome aboard!
 
-We do require that distributions follow our trademark policy https://www.chef.io/trademark-policy/
+Anyone is welcome to make a distribution of Chef’s Open Source products, but must remove Chef trademarks from the code base (see the full [Chef trademark policy](https://www.chef.io/trademark-policy/) for details). 
 
-What does that mean? It means you need to remove enough of the Chef Marks from the source code so as to not confuse users as to the origin of the distribution
+## Why does Chef require removal of Trademarks? 
+
+The purpose of Chef’s Trademark Policy is to avoid any confusion about which distributions come from Chef the company (which are supported, warrantied, and indemnified) and which distributions come from somewhere else (which are not supported, warrantied, or indemnified by Chef). 
+
+## How does this affect you?
 
 If you offer products or services related to Chef products, you may use the Chef name in describing and advertising your product so long as you don't mislead customers into thinking that either your website, service, or product is a Chef Software website, service, or product, or that Chef has any direct relationship with your organization. For example: "Nell's Management Dashboard for Chef Habitat" is okay. "Nell's Chef" is not okay, nor is "Nell's Chef Automate Dashboard".
 
-Additionally, commands used to execute the distribution must be different from those in Chef Software's distribution.  For example, if you create a distribution of Chef, the command to use your distribution's binary cannot be "chef", though it can be something like "kitty" - the goal is prevent confusion about which distributions come from Chef Software and which come from another source.
+Additionally, commands used to execute the distribution - if they include trademarks of Chef - must be different from those in Chef Software's distribution. For example, if you create a distribution of Chef, the command to use your distribution's binary cannot be "chef", though it can be something like "kitty" - the goal is prevent confusion about which distributions come from Chef Software and which come from another source.
```

Comment:
I remember some conclusion in Slack regarding commands, and wrapper like [this](https://gitlab.com/cc-build/outspec/merge_requests/9/diffs#diff-content-88b7c64cc02b2834e36613c781436de35ba3b549) were ok as far I remember. Is it possible to cover this point here too?

Reply:
Fixed!